### PR TITLE
Fixes for chat input (#3412, #3481); animations leftovers removed

### LINKED ISSUE #3412: Tapping outside the commands window does not close it
### Description

*Type*: Bug

*Summary*: This is from `status-everyday` feedback @vkjr has provided cc @denis-sharypin 

#### Expected behavior

when press on a commands list icon, list of commands slides out. Intuitively I want to close it by pressing on the messages area (outside the list of commands)

#### Actual behavior
^ it is not possible, I need to select message edit field again and remove ‘/’ character 

### Reproduction

- Open Status
- Press commands list icon
- Tap outside the commands window

### Additional Information
* Status version: last nightly 248
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by alwx**
Fixes #3412 and #3481.
This PR also removes some no longer needed things like `header-boxes` for suggestions.

### CODE REVIEW

File: src/status_im/chat/views/input/animations/expandable.cljs [hunk 1]
```diff
@@ -5,7 +5,8 @@
             [status-im.ui.components.react :as react]
             [status-im.chat.styles.animations :as style]
             [status-im.chat.styles.input.input :as input-style]
-            [status-im.utils.platform :as platform]))
+            [status-im.utils.platform :as platform]
+            [taoensso.timbre :as log]))
```
Comment (jeluard) [inline src/status_im/chat/views/input/animations/expandable.cljs]: Do we need this?

File: src/status_im/chat/views/input/result_box.cljs [hunk 2]
```diff
@@ -1,23 +1,13 @@
...
-   markup])
-
 (defview result-box-view []
   [markup [:result-box-markup]]
   (when markup
-    [expandable-view {:key           :result-box
-                      :custom-header (box-header/get-header :result-box)}
-     [result-box-container markup]
+    [expandable/expandable-view {:key :result-box}
+     [react/view {:flex 1}
```
Comment (jeluard) [inline src/status_im/chat/views/input/result_box.cljs]: Can we define this in `styles` ?

**Review by jeluard — APPROVED** [review]

**Review by janherich — APPROVED** [review]

File: src/status_im/chat/views/input/result_box.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/chat/views/input/result_box.cljs]: we have it in common styles

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
1. https://github.com/status-im/status-react/issues/3412 is still reproducible on android
2. regarding https://github.com/status-im/status-react/issues/3412 on ios - when i tap outside the command window disappears but `/` symbol is still shown in the chat box. I assume when we close the command window we dont need commands from that moment so when i then tap into the chat box i want to see the keyboard. Please see https://app.testfairy.com/projects/4803590-status/builds/7874853/sessions/13/?accessToken=OEtW9kIUW5XZKkfXHpfEvjdiwV8
@denis-sharypin WDYT? ^
@alwx pelase take a look at those two

**Comment by denis-sharypin** [comment]
@asemiankevich agree with you on the 2 second point. Thanks for cath this :eyes:

**Review by goranjovic — APPROVED** [review]

**Pull request merged**